Backend/src/main/java/lt/vu/ads/models/order/json/OrderView.java

```diff
@@ -22,6 +23,8 @@ public class OrderView {
     private Date estimatedArrivalTime;
     private Date convenientArrivalTimeFrom;
     private Date convenientArrivalTimeTo;
+    private Address sourceAddress;
+    private Address destinationAddress;
```

Comment:
Should use views(dtos), and if full address is not always needed when returning this view, should return just id, that could be used to get address by id afterwards